tests/installation/add_maintenance_repos.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add new add-ons using maintenance test repo URLs.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi 'get_var';
+
+sub run {
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+
+    $testapi::distri->get_add_on_product()->confirm_like_additional_add_on();
+
+    while (defined(my $maintrepo = shift @repos)) {
+        $testapi::distri->get_add_on_product()->accept_current_media_type_selection();
+        $testapi::distri->get_repository_url()->add_repo({url => $maintrepo});
```

Comment:
we are missing `name` parameter, as we wanted to do something similar to what we have in AutoYaST, it is more clear to have the repos with names in the installed system than with some hashes that are created when no name is introduced.

Reply:
This module reads the maintenance repos as they are given in the relevant variables, meaning the repos separated by comas, no name. This is also the case for the original module. How would we insert names there? We would have to change the variable structure and way they are read.